packages/cli/src/services/project.service.ee.ts

```diff
@@ -211,28 +224,80 @@ export class ProjectService {
 
 	async syncProjectRelations(
```

Comment:
TODO: add a test to make sure this throws when used with personal projects or the personalOwner role.

Reply:
done